specification/metadata/metadata.mdpp

```diff
@@ -0,0 +1,6 @@
+# Metadata
+
+The FOCUS specification defines a metadata structure that is to be supplied by data providers to facilitate practioners use of FOCUS data. This meta data includes general information about the data generator and the schema of the FOCUS dataset. FOCUS Metadata SHOULD be provided in a format that is accessible programmatically, such as: a file, website, api, table.
```

Comment:
typo: practioner

Reply:
Question: How are billing data sets and the corresponding metadata correlated?

Reply:
We spoke about this in detail on the TF2 call today. At this stage we are not being prescriptive of the correlation approach but we do see this as a future item for discussion.

Reply:
```suggestion
The FOCUS specification defines a metadata structure that is to be supplied by data providers to facilitate practitioners use of FOCUS data. This meta data includes general information about the data generator and the schema of the FOCUS dataset. FOCUS Metadata SHOULD be provided in a format that is accessible programmatically, such as: a file, website, api, table.
```